### PR TITLE
Editor: Update WordPress packages from Gutenberg 10.5

### PR DESCRIPTION
**Pull request by gziolo**
Trac ticket: https://core.trac.wordpress.org/ticket/52991

First pass, I haven't tested it yet. I plan to take care of it tomorrow.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/blocks/cover.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
```
Comment (youknowriad) [inline src/wp-includes/blocks/cover.php:1]: I have vague memories suggestion that this one was a plugin only thing while waiting for a core change. but it's old, we may have to check the original PR.

The fact that it's an old thing in the plugin that is not in Core suggests that we could avoid it?

Comment (gziolo) [reply src/wp-includes/blocks/cover.php:1]: I see that @mtias authored https://github.com/WordPress/gutenberg/pull/22346 that added this change.

It's already in WP 5.6:

https://github.com/WordPress/wordpress-develop/blob/9ec7be02d1d5985e35caba72de784a651a0dae9d/src/wp-includes/kses.php#L406

So we should omit it here and completely remove from the Gutenberg plugin and WordPress packages.

Comment (youknowriad) [reply src/wp-includes/blocks/cover.php:1]: Makes sense 👍

Comment (gziolo) [reply src/wp-includes/blocks/cover.php]: Fixed in https://github.com/WordPress/wordpress-develop/pull/1226/commits/d8becf7e4e99abb80fb6fcbb9298f51a180134a0.

**Comment by gziolo** [comment]
There is one more thing left that I almost missed. For the File block, we need to add an entry point for the JS file used on the frontend.

File: src/wp-includes/blocks/cover.php (hunk 1 above)
Comment (gziolo) [reply src/wp-includes/blocks/cover.php]: PR in Gutenberg: https://github.com/WordPress/gutenberg/pull/31503.

**Comment by gziolo** [comment]
> There is one more thing left that I almost missed. For the File block, we need to add an entry point for the JS file used on the frontend and exposed under the `wp-block-library-file` script handle.

It's a more complex task than I anticipated. Let's land the changes include already and tackle changes to the build system separately.

**Comment by gziolo** [comment]
Landed with https://core.trac.wordpress.org/changeset/50824.

**Pull request closed without merging**